lineage/figures/figureS12.py

```diff
@@ -89,16 +85,18 @@ def accuracy():
     for indx, num in enumerate(num_lineages):
         population3 = []
         for _ in range(10):
+            
             good2go = False
             while not good2go:
                 tmp_lineage3 = LineageTree.init_from_parameters(pi3, T3, E3, desired_num_cells=min_desired_num_cells, censor_condition=3, desired_experiment_time=experiment_times[indx])
                 good2go = lineage_good_to_analyze(tmp_lineage3)
             
             population3.append(tmp_lineage3)
     
-    tree_obj, predicted_states, LL = run_Analyze_over(population3, 3)
-    results_dict = Results(tree_obj, predicted_states, LL)
-    accuracy_3 = results_dict["accuracy_after_switching"]
+    # Adding populations into a holder for analysing
+        list_of_populations2.append(population2)
+
+    cell_number_x2, _, accuracy2_after_switching, _, _, _ = commonAnalyze(list_of_populations2)
```

Comment:
You should change the names of `cell_number_x2` to `cell_number_x3` for 3 state and for 4 state x4, as well.